site/gatsby-site/src/hooks/useToast.js

```diff
@@ -13,25 +13,25 @@ export const SEVERITY = {
     key: 'success',
     icon: faCheckCircle,
     faClass: 'far fa-check-circle fa-lg',
-    color: '#43a047',
```

Comment:
Can this be set somewhere else using classes? It would make theming easier, and make it easier to have an eventual dark mode.

Reply:
If the colors already exist in tailwind, they could be applied as classes after faClass like:
`faClass: 'far fa-check-circle fa-lg',
bgColor:'bg-green-700'
`

And then used as:
`<ToastBodyContent className={severity.bgColor}>`

Reply:
@clari182 When I try this, I just get a white background. When I inspect the element, it shows class="… bg-green-700 …", but the browser doesn't apply any rules based on it. Do you have an idea why that might be? Is that because it's now inside a ".bootstrap"?

Reply:
@lmcnulty Do you have the latest staging changes?
This could be happening because I added some changes in the tailwind config that are on staging now.
- the prefix `tw-` has been removed
- the property "important" has been added:   `important: true` as first prop in `tailwind.config.js` inside module.exports
Something else that could be happening is if you're using the class dynamically. Tailwind doesn't allow classes to be used in that way like `bg-${color}` for example.
Let me know if this helps.

Reply:
@clari182 I thought those changes were already merged, but merging staging again seems to have fixed it.